### ISSUE TITLE
Should we delete the contents of this repo and start over?

### ISSUE THREAD

**Issue by tenhagena** [funstuff234/TaskManagerAndroid#6]

**Comment by cmonteiro128** [comment]
No leave this, we'll make a new repo once we finalize our new idea.

**Comment by tenhagena** [comment]
OK fun stuff! then why do we still have this repo?

**Comment by poliaf** [comment]
We may continue work on this project eventually, but for now we should just keep the repo as-is while we work on other projects. There is no harm in keeping it.

**Comment by ghost** [comment]
Yeah, we'll probably decide on Monday what we want to do exactly.
